pkg/gds/store/store.go

```diff
@@ -52,6 +52,7 @@ const (
 	NamespaceCertReqs = "certreqs"
 	NamespaceReplicas = "peers"
 	NamespaceIndices  = "index"
+	NamespaceSequence = "sequence"
```

Comment:
This is an artifact of older code, but leaving it in if we want to store ints later.